skbio/io/format/genbank.py

```diff
@@ -435,20 +422,38 @@ def _serialize_single_genbank(obj, fh):
     Always write it in NCBI canonical way:
     1. sequence in lowercase
     2. 'u' as 't' even in RNA molecules.
+
+    Parameters
+    ----------
+    obj : Sequence or its child class
+
     '''
+    # write out the headers
     md = obj.metadata
     for header in _HEADERS:
         if header in md:
             serializer = _SERIALIZER_TABLE.get(
                 header, _serialize_section_default)
-            out = serializer(header, md[header])
-            # test if 'out' is a iterator.
-            # cf. Effective Python Item 17
-            if iter(out) is iter(out):
-                for s in out:
+            if header == 'FEATURES':
+                indent = 21
```

Comment:
Can you note where this magic number came from, or (better) remove the magic number in favor of something more self-documenting?